lib/jekyll.rb

```diff
@@ -172,6 +172,7 @@ def sanitized_path(base_directory, questionable_path)
 require_all 'jekyll/commands'
 require_all 'jekyll/converters'
 require_all 'jekyll/converters/markdown'
+require_all 'jekyll/converters/smartypants'
```

Comment:
Line 173 included `jekyll/converters/smartypants.rb` but not `jekyll/converters/smartypants/kramdown_parser.rb`